src/meeting/meet/htmlCleaner.ts

```
@@ -0,0 +1,283 @@
+            ;(window as any).htmlCleanerObserver = observer
+            console.log('[Meet] HTML provider complete')
+        }, this.recordingMode)
+    }
+
+    public async stop(): Promise<void> {
+        console.log('[Meet] Stopping HTML cleaner')
+        
+        await this.page.evaluate(() => {
+            if ((window as any).htmlCleanerObserver) {
+                (window as any).htmlCleanerObserver.disconnect()
+                delete (window as any).htmlCleanerObserver
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)